tests/grpc/test_interceptor.py

```diff
@@ -0,0 +1,129 @@
+import pytest
```

Comment:
thanks for adding these tests. ideally we also should test other metrics / errors but can happen as a follow up PR.